Source/DafnyRuntime/DafnyRuntimeDafny/src/dafnyRuntime.dfy

```diff
@@ -376,7 +376,7 @@ abstract module {:options "/functionSyntax:4"} Dafny {
       requires inv(t)
   }
 
-  trait {:extern} Sequence<T> {
+  trait {:extern} Sequence<T(==)> {
```

Comment:
Note from offline conversation: this isn't actually true in general, so the better fix is to remove the `UniqueElements()` function and workaround that instead.